### PR TITLE
Escanciu Village

### PR DESCRIPTION
**Pull request by vtange**
**to IMC:** Found one unmarked {memo X} line. QUEST200 Line 429 and QUEST100 Line 1931 are identical but the game uses QUEST100 Line 1931.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1347,8 +1347,8 @@ QUEST_LV_0100_20150317_001346	{memo X}
 QUEST_LV_0100_20150317_001347	I knew you are not ordinary.{nl}Thanks for your support.
 QUEST_LV_0100_20150317_001348	I want to take a look at it more closely, but there are too many interruptors.{nl}Can you protect me from those monsters for a while?
 QUEST_LV_0100_20150317_001349	I never knew that I would be able to see these beautiful patterns.{nl}When you go to Garden of Pillar, there may be more beautiful patterns. Follow me.
-QUEST_LV_0100_20150317_001350	It seems that wandering spirits want to give the rewards to someone in Fedimian.
-QUEST_LV_0100_20150317_001351	After I finish the work, Unicorn appeared.
+QUEST_LV_0100_20150317_001350	$It seems that Wandering Spirit want to give the reward to someone in Fedimian.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: that Wandering Spirit want -> that the Wandering Spirit wants

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1960,21 +1960,21 @@ QUEST_LV_0100_20150317_001959	There are many artifacts here that praise the hero
...
-QUEST_LV_0100_20150317_001974	Please give this crutches which I made myself to Cahill.{nl}Where was Cahill hurt his ankle..
-QUEST_LV_0100_20150317_001975	Before I change my mind, get outta here.
-QUEST_LV_0100_20150317_001976	Crutches from Moze..{nl}I really don't know what he is thinking.
-QUEST_LV_0100_20150317_001977	I didn't say it's over.{nl}Please get me the leather of Wing Frogs near this camp.{nl}
+QUEST_LV_0100_20150317_001963	$There was a priest called Maven in the past.{nl}He is the person who built the great cathedral.{nl}
+QUEST_LV_0100_20150317_001964	$Maven lived for a long time and had three disciples.{nl}The first one was Ruklys and the second one was Lydia Schaffen.{nl}
+QUEST_LV_0100_20150317_001965	$And when Lydia Schaffen grew old, Agailla Flurry became the last disciple.{nl}As the other two disciples were, Agailla Flurry was a great person.
+QUEST_LV_0100_20150317_001966	$She learned all magics that existed then.{nl}She also built Mage Tower. She must've been amazing.
+QUEST_LV_0100_20150317_001967	$Our old man always complains.{nl}Even when I give him the best things to him, he complains.{nl}
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: "Even when I give him the best things to him" is redundant.

Can be "Even when I give him the best things"
or
"Even when I give the best things to him"

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -1960,21 +1960,21 @@ QUEST_LV_0100_20150317_001959	There are many artifacts here that praise the hero
...
+QUEST_LV_0100_20150317_001968	$He always complains, so I don't want to help him.{nl}Sorry, but can you tell that old man there how I feel?
+QUEST_LV_0100_20150317_001969	$When he first came here, he had a hard time adapting to the new environment.{nl}I used to cook him food and make him clothes..
+QUEST_LV_0100_20150317_001970	$Why didn't Kayhill come here and why you saying that?{nl}Well, it doesn't matter. Are you finished?
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: why you saying that -> why are you saying that

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -1960,21 +1960,21 @@ QUEST_LV_0100_20150317_001959	There are many artifacts here that praise the hero
...
+QUEST_LV_0100_20150317_001971	$Hey. Since you came, please do some errands.{nl}You see those lumbers over there? Please bring me the good ones among them.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: "Hey. Since you came, please do some errands for me.{nl}You see that pile of lumber over there? Bring me the good ones among them."
?

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -1960,21 +1960,21 @@ QUEST_LV_0100_20150317_001959	There are many artifacts here that praise the hero
...
+QUEST_LV_0100_20150317_001972	$Kayhill is helpful. {nl}Except for his complaints..
+QUEST_LV_0100_20150317_001973	$What! Can't you find more decent ones?{nl}Well.. that's still okay..
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: What! -> What?!

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -1960,21 +1960,21 @@ QUEST_LV_0100_20150317_001959	There are many artifacts here that praise the hero
...
+QUEST_LV_0100_20150317_001974	$Please give these crutches I made to Kayhill.{nl}Now where oh where did Kayhill hurt his ankle..
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: Comma after the first where.

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -1960,21 +1960,21 @@ QUEST_LV_0100_20150317_001959	There are many artifacts here that praise the hero
...
+QUEST_LV_0100_20150317_001975	$Before I change my mind, get outta here.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: It'd sound better if it was "Get outta here before I change my mind." IMO

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -6101,37 +6101,37 @@ QUEST_LV_0100_20150317_006100	{memo X}Talk to the item merchant
...
+QUEST_LV_0100_20150317_006120	$Kayhill is grumbling about something. Talk to him.
+QUEST_LV_0100_20150317_006121	$Talk to Moje
+QUEST_LV_0100_20150317_006122	$Kayhill is complaining about Moje who is bugging him. Tell Moje how Kayhill feels.
+QUEST_LV_0100_20150317_006123	$Give lumber to Moje
+QUEST_LV_0100_20150317_006124	$Moje is not interested in Kayhill and tells you to just do him a favor. Help him fetch some good lumber beside his house.
+QUEST_LV_0100_20150317_006125	$Looks like you've chosen good lumber. Give them to Moje.
+QUEST_LV_0100_20150317_006126	$Talk to Moje Diesel
+QUEST_LV_0100_20150317_006127	$You thought you chose pretty good lumber, but Moje is not satisfied. Talk to him.
+QUEST_LV_0100_20150317_006128	$Get the tough leather of Flying Frogs
+QUEST_LV_0100_20150317_006129	$Moje is still looking to use you some more. Get tough leather from Flying Frog near the campsite.
```
Comment (ghost) [inline QUEST_LV_0100.tsv]: Flying Frog -> Flying Frogs
Same with line 6131

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -3642,32 +3642,32 @@ QUEST_LV_0100_20150317_003641	Bishop's Dream (1)
...
-QUEST_LV_0100_20150317_003652	{memo X}Notice/scroll/Defeat the Zolem and collect Zolem Demon Stones
+QUEST_LV_0100_20150317_003652	{memo X}Notice/scroll/Defeat the Zolrem and collect Zolrem Demon Stones
 QUEST_LV_0100_20150317_003653	{memo X}Working/SITGROPE/2/TRACK_BEFORE/None
-QUEST_LV_0100_20150317_003654	The Last Order (3)
+QUEST_LV_0100_20150317_003654	$Final Mission (3)
 QUEST_LV_0100_20150317_003655	{memo X}Working/MAKING/3/TRACK_BEFORE/None
 QUEST_LV_0100_20150317_003656	Carry out the Tree Dale plan
 QUEST_LV_0100_20150317_003657	Need more preparations
-QUEST_LV_0100_20150317_003658	True Feeling of Mose (1)
+QUEST_LV_0100_20150317_003658	$Moje's True Feelings (1)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Why the change to Moje?

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -3642,32 +3642,32 @@ QUEST_LV_0100_20150317_003641	Bishop's Dream (1)
...
 QUEST_LV_0100_20150317_003659	Tell about the feelings of the town young man
 QUEST_LV_0100_20150317_003660	Better do that yourself
 QUEST_LV_0100_20150317_003661	About Agaila Flurry
 QUEST_LV_0100_20150317_003662	{memo X}Telling how Kayhill feels/TALK/2/TRACK_BEFORE/None
-QUEST_LV_0100_20150317_003663	True Feeling of Mose (2)
+QUEST_LV_0100_20150317_003663	$Moje's True Feelings (2)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Why the change to Moje?

File: QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -3642,32 +3642,32 @@ QUEST_LV_0100_20150317_003641	Bishop's Dream (1)
...
 QUEST_LV_0100_20150317_003664	Bring the woods
 QUEST_LV_0100_20150317_003665	Not an appealing attitude so just ignore
 QUEST_LV_0100_20150317_003666	{memo X}Pick up woods/#SITGROPESET2/2/TRACK_BEFORE/None
-QUEST_LV_0100_20150317_003667	True Feeling of Mose (3)
+QUEST_LV_0100_20150317_003667	$Moje's True Feelings (3)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Why the change to Moje?

File: QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -3642,32 +3642,32 @@ QUEST_LV_0100_20150317_003641	Bishop's Dream (1)
...
 QUEST_LV_0100_20150317_003668	Get the leather
 QUEST_LV_0100_20150317_003669	Not an appealing attitude so just ignore
-QUEST_LV_0100_20150317_003670	True Feeling of Mose (4)
+QUEST_LV_0100_20150317_003670	$Moje's True Feelings (4)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Why the change to Moje?

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -5406,12 +5406,12 @@ QUEST_LV_0100_20150317_005405	Defeat the Necroventer out of control
 QUEST_LV_0100_20150317_005406	Necroventer was summoned as a result of necromancing experiment. But it is out of control. Defeat it before it causes too much harm. 
 QUEST_LV_0100_20150317_005407	The experiment failed but you managed to defeat the Necroventer. Tell Drasius about it. 
 QUEST_LV_0100_20150317_005408	Defeat Necroventer
-QUEST_LV_0100_20150317_005409	Talk to the wandering soul
+QUEST_LV_0100_20150317_005409	$Talk to the Wandering Soul
 QUEST_LV_0100_20150317_005410	{memo X}
 QUEST_LV_0100_20150317_005411	{memo X}Return the trophy of war
-QUEST_LV_0100_20150317_005412	The wandering soul wants you to return the unicorn horn which was a trophy of war obtained by defeating the unicorn in Fedimian to Fedimian military official. Go find the military official. 
+QUEST_LV_0100_20150317_005412	$The Wandering Soul wants you to return the Unicorn Horn you obtained as a trophy of war to the Fedimian military official in Fedimian. Go find the military official.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Go find the military official. -> Find the military official.

File: ETC.tsv [hunk 14]
```diff
@@ -723,7 +723,7 @@ ETC_20150317_000722	Black Large Slime
 ETC_20150317_000723	Fire Golem
 ETC_20150317_000724	Warmed Up Gorgon Golem
 ETC_20150317_000725	Noble Tomblord
-ETC_20150317_000726	Blue Zolem
+ETC_20150317_000726	Blue Zolrem
```
Comment (Sunaries) [inline ETC.tsv]: Why did you change it to Zolrem? Zolem works.

Comment (vtange) [inline ETC.tsv]: I changed it to keep things consistent. In Escanciu you find monsters called "Zolrem" which are refered to as Zolems, so I changed everything to "Zolrem". Should I change it all to "Zolem"?

File: QUEST_LV_0100.tsv (hunk 9 above)
Comment (vtange) [inline QUEST_LV_0100.tsv]: IIRC the NPC name in QUEST200.tsv was  "Sir Moje". Should it be "Sir Mose"?

File: ETC.tsv (hunk 14 above)
Comment (Sunaries) [inline ETC.tsv]: Yeah, change it back to Zolem. Keep in mind that there are a lot of inconsistencies.

File: QUEST_LV_0100.tsv (hunk 9 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Moses/Mose.

File: ETC.tsv [hunk 15]
```diff
@@ -7844,7 +7844,7 @@ ETC_20150317_007844	Get the keepsake
 ETC_20150317_007845	{memo X}Victim Liberation
 ETC_20150317_007846	{memo X}Truth transmission of Ruklys
 ETC_20150317_007847	{memo X}Use a Petrifaction Fusion
-ETC_20150317_007848	Cordiality of Moje Dcell (3)
+ETC_20150317_007848	Cordiality of Moje Diesel (3)
```
Comment (Sunaries) [inline ETC.tsv]: Moses

File: QUEST_LV_0100.tsv [hunk 16]
```diff
@@ -2371,7 +2371,7 @@ QUEST_LV_0100_20150317_002370	$To the Scout (1)
 QUEST_LV_0100_20150317_002371	$I'm just here to tell the troops to assemble
 QUEST_LV_0100_20150317_002372	$Quickly run away
 QUEST_LV_0100_20150317_002373	$To the Scout (2)
-QUEST_LV_0100_20150317_002374	$I'll find the missing luggage
+QUEST_LV_0100_20150317_002374	$I'll find the missing bag
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Soukyuu has this covered already. I believe he made a more fitting term for it, "belongings". 
Check his PR.

File: QUEST_LV_0100.tsv [hunk 17]
```diff
@@ -8095,7 +8095,7 @@ QUEST_LV_0100_20150908_008094
 QUEST_LV_0100_20150908_008095	
 QUEST_LV_0100_20150908_008096	
 QUEST_LV_0100_20150918_008097	This place became dangerous because of monsters emerging since Medis Diena.{nl}Quickly return while you can, because Kepa's will soon be marching in.
-QUEST_LV_0100_20150918_008098	You were telling that we need to assemble, right?{nl}Please search my luggage first and return quickly, so I'll go as soon as I can.
+QUEST_LV_0100_20150918_008098	You were telling that we need to assemble, right?{nl}Please search my bag first and return quickly, so I'll go as soon as I can.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Soukyuu has this covered already. I believe he made a more fitting term for it, "belongings". 
Check his PR.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:
